src/multiplayer-servers/getting-started/setup-third-party-oauth.md

```diff
@@ -0,0 +1,150 @@
+# Setup Third-Party OAuth Integration
+
+## Overview
+
+GameFabric supports Third-Party OAuth Integration with every OAuth-compatible Identiy Provider, allowing you to reuse your existing authentication flows, without giving GameFabric access to your sensitive authentication data.
+
+## Prerequisites
+
+Before setting up a Third-Party OIDC Provider, ensure the following conditions are met:
+
+- You have access to an active GameFabric installation with permissions to modify OAuth Providers or the default user. 
+- You have an OIDC ([OpenID Connect](https://openid.net/developers/how-connect-works/)) provider, including a **Client ID** and **Client Secret**.
+
+## Setting up a new OIDC Provider
+
+Navigate to the "Access Management" section from the Sidebar Menu, and then into the "OIDC Providers" page. You may add a new OIDC Provider via the **Add Provider** button.
+
+![Add Provider Screenshot](images/authentication/create-oidc-provider-main-screenshot.png)
+
+For outstanding providers such as Microsoft ENTRA or Google Identity Platform; following instructions in [Preliminary Configurations](#preliminary-configurations) are sufficient to complete the integration.
+
+If you are using a non-standardized OIDC Provider, or would like to further customize the authentication flow (modifying prompt behaviour of your OIDC Provider, etc.) please refer to [Advanced Configurations](#advanced-configurations).
+
+### Preliminary configurations
+
+On the "General" step, enter a desired **ID** and **Display Name** for your provider. The OIDC Provider you are about to add will be saved under this **ID** and **Display Name** on GameFabric.
+
+![Provider Details Screenshot](images/authentication/create-oidc-provider-firststep-screenshot.png)
+
+On the "Provider" step, you need to specify the **Issuer URL** of your OIDC provider. This URL points to the OIDC Discovery Document of that particular provider, and GameFabric uses this Discovery Document for integrating with that provider.
```

Comment:
There is a note on this. If memory serves, the issuer URL should not include the "well-known" (`/.well-known/openid-configuration`) portion of the final url. Doing so will make it fail. This was found out the hard way.